functionary/ui/templates/forms/scheduled_task/scheduled_task_edit.html

```diff
@@ -1,233 +1,119 @@
-                        <div class="field">
-                            <div class="columns is-8">
-                                <div class="column is-one-third">
-                                    <label for="{{ form.description.id_for_label }}">
-                                        <b {% if form.description.errors %}class="errors"{% endif %}
-                                           class="title is-size-4">{{ form.description.label }}</b>
-                                        <span class="is-small has-text-grey-light">{{ form.description.label_suffix }}</span>
-                                        &nbsp;&nbsp;&nbsp;{{ form.description.help_text }}
-                                    </label>
-                                    <div class="column is-one-half p-0 is-italic subtitle is-size-6">
-                                        <p>Give a description for your scheduled task.</p>
-                                    </div>
-                                    <div class="errorlist">{{ form.description.errors }}</div>
-                                </div>
-                                <div class="column">
-                                    <div class="control">{{ form.description }}</div>
-                                </div>
-                            </div>
-                        </div>
-                        <hr/>
-                        <!-- Function row/field -->
-                        <div class="field">
-                            <div class="columns">
-                                <div class="column is-one-third">
-                                    <label for="{{ form.function.id_for_label }}">
-                                        <b class="title is-size-4">{{ form.function.label }}</b>
-                                        <span class="is-small has-text-grey-light">{{ form.function.label_suffix }}</span>
-                                        &nbsp;&nbsp;&nbsp;{{ form.function.help_text }}
-                                    </label>
-                                    <div class="column p-0 is-italic subtitle is-size-6">
-                                        <p>Select which function you want to schedule.</p>
-                                    </div>
-                                    <div class="errorlist">{{ form.function.errors }}</div>
-                                </div>
-                                <div class="column is-one-third">
-                                    {% if update %}
-                                        <div class="control">{% render_field form.function disabled=True %}</div>
-                                    {% else %}
-                                        <div class="control">{{ form.function }}</div>
+                        <div>
+                            <strong class="fs-4"><i class="fa fa-calendar fa-sm fa-fw pe-1"></i>Crontab Schedule</strong>
+                            <div class="ps-4 py-2 vstack gap-4">
```

Comment:
We probably want to do something other than `vstack` here.  The input fields here are going to take a max of like 4 characters, so having them take the entire width of the form feels excessive.  Just switching to `hstack` seems okay, but has two problems of its own:

* The field labels get cutoff
* The error message placement no longer really works

Those both feel like solvable problems though and it would clean up the page a lot I think.